packages/react-examples/.storybook/main.js

```diff
@@ -3,15 +3,19 @@ import custom from '@fluentui/scripts/storybook/webpack.config';
 import * as path from 'path';
 import * as webpack from 'webpack';
 
+const packageName = path.basename(process.cwd());
```

Comment:
This only works when run from the root of a package, is there another more safe way to do this?